exchanges/mock/common_test.go

```diff
@@ -105,14 +63,10 @@ func TestDeriveURLValsFromJSON(t *testing.T) {
 	}
 
 	payload, err := json.Marshal(test1)
-	if err != nil {
-		t.Error("marshal error", err)
-	}
+	require.NoError(t, err, "json.Marshal must not error")
 
 	_, err = DeriveURLValsFromJSONMap(payload)
-	if err != nil {
-		t.Error("DeriveURLValsFromJSON error", err)
-	}
+	assert.NoError(t, err, "DeriveURLValsFromJSON error should not error")
```

Comment:
🥃 Just an aside that the test comment is optional here, because it's in a unit test of the same name